### PR TITLE
Documenation Update: Adding CodeOwner and Couchbase user documentation

### PR DESCRIPTION
**Pull request by ankurtyagi-sdk**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build was run locally and any changes were pushed
- [ ] Lint has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@ankurtyagi-sdk @pratap-akhand @varunshrivastava2502
```
Comment (ankursarin) [inline CODEOWNERS:1]: suggestion: move to a `.github` directory?

Comment (ankurtyagi-sdk) [reply CODEOWNERS]: @ankursarin we have moved the .docx document to markdown (.md) file. All images inside the user documentation have been linked with .md file and the images are stored in "images" folder on repository base location.

Also moved CODEOWNERS file to .github directory.

Comment (ankursarin) [reply CODEOWNERS]: sounds good

File: CouchbaseUserDocumentation.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+=======
+
+This document will help to install configure couchbase edsi
+
+a)  .....
+
+b)  .....
+
+c)  .....
+
```
Comment (ankursarin) [inline CouchbaseUserDocumentation.md:54]: needs to be updated?

**Pull request merged**